packages/expo/package.json

```diff
@@ -83,6 +83,7 @@
     "typescript": "*"
   },
   "peerDependencies": {
+    "@clerk/expo-passkeys": ">=0.0.6",
```

Comment:
As discussed, since this is an optional peer, we should add `peerDependenciesMeta` declaration